contributors/devel/pull-requests.md

```diff
@@ -142,6 +143,14 @@ Either the [on call contributor](on-call-rotations.md) will manage the merge que
 
 That's the last step. Your PR is now merged.
 
+## Marking Unfinished Pull Requests
+
+If you want to solicit reviews before the implementation of your pull request is complete, you should mark your pull request as a work in progress to ensure that the merge queue does not pick it up and attempt to merge it. You may prefix your pull request title with `WIP` or `[WIP]` to indicate this.
+
+The GitHub robots will add and remove the `do-not-merge/work-in-progress` label as you edit your title. While the label is present, your pull request will not be considered for merging.
+
+If the pull request is not a work in progress, but you still want to hold the merge queue from merging it, use the `/hold` comment command to do so. When you're ready for the queue to consider your PR for merges again, use the `/hold cancel` comment command.
```

Comment:
Can we please put this at https://github.com/kubernetes/test-infra/blob/master/commands.md as well?

Also my nit suggestion would be to prioritize this recommendation (as it is more flexibile) over WIP

Reply:
The PR adding `/hold` updates that page, but has not merged yet.